src/pushStatusHandler.js

```diff
@@ -36,7 +36,7 @@ export default function pushStatusHandler(config) {
       expiry: body.expiration_time,
       status: "pending",
       numSent: 0,
-      pushHash: md5Hash(data.alert || ''),
+      pushHash: md5Hash(JSON.stringify(data.alert) || ''),
```

Comment:
in the case alert is a string, that will wrap the string into another string, including the quotes...
This should be updated to:

```
let pushHash;
if (typeof data.alert === 'string') {
  pushHash = md5Hash(data.alert);
} else if (typeof data.alert === 'object') {
  pushHash = md5Hash(JSON.stringify(data.alert));
} else {
   pushHash = "d41d8cd98f00b204e9800998ecf8427e"; // empty string hash
}
```